src/indra_cogex/client/queries.py

```diff
@@ -840,6 +850,32 @@ def get_stmts_for_stmt_hashes(
     return list(stmts.values())
 
 
+def _get_mesh_child_terms(client: Neo4jClient, meshid: Tuple[str, str]) -> Set[str]:
```

Comment:
Or did I misunderstand? Did you propose to rename all the id inputs like this `<type>_id` -> `<type>_curie`?

Reply:
At minimum I would suggest `<type>_id` is better than `<type>id`.

Maybe `<type>_curie` is more descriptive than `<type>_id`, but on second thought, this would mean we would have to change other code too.